app/components/press-release/files-modal.hbs

```diff
@@ -0,0 +1,29 @@
+<Auk::Modal>
+  <Auk::Modal::Header
+    @title="Bijlage toegoegen"
+    @onClose={{this.closeModal}}
+  />
+  <Auk::Modal::Body>
+    <div class="auk-form-group-layout auk-form-group-layout--standard">
+      <div class="auk-form-group">
+        <Auk::Label>
+          Bijlage
+        </Auk::Label>
+          <Auk::FileUploader
+            @fileQueueName={{@fileQueueName}}
+            @multiple={{false}}
+            @onUpload={{this.addFile}}
+          />
+      </div>
+    </div>
+  </Auk::Modal::Body>
+  <Auk::Modal::Footer @onCancel={{this.closeModal}}>
+    <Auk::Button
+      @skin="primary"
+      @loading={{@loading}}
```

Comment:
Dit heeft geen effect want `@loading` wordt niet meegegeven.
Hier wil je vermoedelijk wachten tot `@onUpload` voltooid is. Dat kan je doen door `@onUpload` te wrappen in een ember-concurrency task in deze component.